src/11.numberformat.js

```diff
@@ -66,12 +66,64 @@ defineProperty(Intl.NumberFormat, 'prototype', {
     writable: false
 });
 
+/*
+ * @spec[tc39/ecma402/master/spec/numberformat.html]
+ * @clause[sec-setnumberformatdigitoptions]
+ */
+export function /*11.1.1 */SetNumberFormatDigitOptions (intlObj, options, mnfdDefault) {
+    // 1. Assert: Type(intlObj) is Object and intlObj.[[initializedIntlObject]] is true.
+
+    // 2. Assert: Type(options) is Object.
+
+    // 3. Assert: type(mnfdDefault) is Number.
+
+    // 4. Let mnid be ? GetNumberOption(options, "minimumIntegerDigits,", 1, 21, 1).
+    let mnid = GetNumberOption(options, 'minimumIntegerDigits', 1, 21, 1);
+
+    // 5. Let mnfd be ? GetNumberOption(options, "minimumFractionDigits", 0, 20, mnfdDefault).
+    let mnfd = GetNumberOption(options, 'minimumFractionDigits', 0, 20, mnfdDefault);
+
+    // 6. Let mxfd be ? GetNumberOption(options, "maximumFractionDigits", mnfd, 20).
+    let mxfd = GetNumberOption(options, 'maximumFractionDigits', mnfd, 20);
+
+    // 7. Let mnsd be ? Get(options, "minimumSignificantDigits").
+    let mnsd = options.minimumSignificantDigits;
+
+    // 8. Let mxsd be ? Get(options, "maximumSignificantDigits").
+    let mxsd = options.maximumSignificantDigits;
+
+    // 9. Set intlObj.[[minimumIntegerDigits]] to mnid.
+    intlObj['[[minimumIntegerDigits]]'] = mnid;
+
+    // 10. Set intlObj.[[minimumFractionDigits]] to mnfd.
+    intlObj['[[minimumFractionDigits]]'] = mnfd;
+
+    // 11. Set intlObj.[[maximumFractionDigits]] to mxfd.
+    intlObj['[[maximumFractionDigits]]'] = mxfd;
+
+    // 12. If mnsd is not undefined or mxsd is not undefined, then
+    if (mnsd !== undefined || mxsd !== undefined) {
```

Comment:
@zbraniecki we changed this one a little bit I think, can you update the implementation?